configure.ac

```diff
@@ -137,7 +137,7 @@ AC_MSG_CHECKING([for jar installation path])
 AC_ARG_WITH(jardir,
             AS_HELP_STRING([--with-jardir],
                            [jar file installation path [LIBDIR/jni]]),
-            jardir=${withval}, jardir=${libdir}/jni)
+            jardir=${withval}, jardir=${prefix}/lib/jni)
```

Comment:
${libdir} expanded to ${exec_prefix}/lib, while this expands to the absolute path of /lib. I looked around a bit and it seemed like we would need to set exec_prefix to prefix? In any case I didn't see much difference between that change and this, so I went with this instead.